src/index.css

```diff
@@ -1,13 +1,12 @@
-body {
+* {
+  padding: 0;
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+  box-sizing: border-box;
+  font-family: monospace;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
-    monospace;
+body {
+  margin: 0;
+  color: white;
+  background-color: #fd5193;
 }
```

Comment:
- [ ] Good job so far cause your application is very well implemented :heavy_check_mark:  But you have to work on your app design, it is a bit different to the original design :cry:  Please I would like to recommend you follow these [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) by using the same layout, position, and other elements to make it similar to the original design. :+1: 

#### Original Design
Please check the screenshots to see the differences  below :arrow_down: 
<img width="421" alt="187742519-59d05e87-670a-4fd1-a83b-dfb4e3cf6644" src="https://user-images.githubusercontent.com/6615394/188026464-3395d9fa-dae8-4f20-adc7-3fdfe97876cb.png">

#### Your design

![Screenshot from 2022-09-01 22-59-00](https://user-images.githubusercontent.com/6615394/188026630-edc80640-5e63-4c0d-9223-2c01d599f126.png)


![Screenshot from 2022-09-01 22-59-14](https://user-images.githubusercontent.com/6615394/188026629-24ceefba-8db7-4113-a282-d3c18182fb63.png)